### PR TITLE
wip

### PR DESCRIPTION
**Pull request by danlong005**
Adding a destroy so that I can deactivate subscriptions.

### CODE REVIEW

File: lib/cashbox/concern/persistable.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ module Persistable
       extend SingleForwardable
 
       def_single_delegators :repository, :where, :all, :first, :find
-      def_instance_delegator :repository, :save
+      def_instance_delegators :repository, :save, :destroy
```
Comment (maulin) [inline lib/cashbox/concern/persistable.rb]: add a test for this guy too

File: lib/cashbox/repository.rb [hunk 2]
```diff
@@ -34,6 +34,11 @@ def save
       cast(request.response)
     end
 
+    def destroy
+      request = Cashbox::Request.new(:post, "#{route(@instance.vid)}/actions/cancel")
```
[the remark this answers is not in the document]
Comment (danlong005) [reply lib/cashbox/repository.rb]: double checking... I am thinking that the only thing we are going to be able to cancel is the subscription.

[the remark this answers is not in the document]
Comment (danlong005) [reply lib/cashbox/repository.rb]: we can cancell transactions and subscriptions... and they both use this url extension.

File: lib/cashbox/concern/persistable.rb (hunk 1 above)
Comment (danlong005) [reply lib/cashbox/concern/persistable.rb]: done

**Pull request merged**